CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ Mapbox welcomes participation and contributions from everyone.
 
 ## Bug fixes 🐞
 * Enable two finger pan gesture. ([#1280](https://github.com/mapbox/mapbox-maps-android/pull/1280))
+* Fix a bug that scale bar is shorter than it should be; trigger `invalidateScaleBar` while updating settings to make scale keep the latest status. ([#1336](https://github.com/mapbox/mapbox-maps-android/pull/1336))
```

Comment:
Imo too verbose, `Fix a bug that scale bar is shorter than it should be` is good enough, rest details are in the PR

Reply:
I'm using `;` to divide reset is to fix https://github.com/mapbox/mapbox-maps-android-internal/issues/1091

Reply:
Ok, got it!